src/components/editor/GroupChoiceModal.jsx

```diff
@@ -4,31 +4,32 @@ import React, { useState } from 'react'
 import { useSelector, useDispatch } from 'react-redux'
 import PropTypes from 'prop-types'
 import { MultiSelect } from 'react-multi-select-component'
-import Config from 'Config'
 import { hideModal } from 'actions/modals'
 import { resourceEditErrorKey } from './Editor'
 import { selectModalType } from 'selectors/modals'
 import { saveNewResource } from 'actionCreators/resources'
 import ModalWrapper, { useDisplayStyle, useModalCss } from '../ModalWrapper'
 import { selectCurrentResourceKey } from 'selectors/resources'
 import { selectGroups } from 'selectors/authenticate'
+import { groupsInSinopia } from 'selectors/groups'
+import { groupNameFromGroup } from 'utilities/Utilities'
+
 
 const GroupChoiceModal = () => {
   const resourceKey = useSelector((state) => selectCurrentResourceKey(state))
   const modalType = useSelector((state) => selectModalType(state))
   const userGroupIds = useSelector((state) => selectGroups(state))
   const [ownerGroupId, setOwnerGroupId] = useState(userGroupIds[0])
   const [editGroupValues, setEditGroupValues] = useState([])
-  const groupMap = Config.groupsInSinopia
   const show = modalType === 'GroupChoiceModal'
   const dispatch = useDispatch()
 
   // TODO: Handle owner group in edit groups. Should be selected and disabled.
 
-  const ownerGroupOptions = userGroupIds.map((groupId) => <option key={groupId} value={ groupId }>{ groupMap[groupId] || groupId }</option>)
+  const ownerGroupOptions = userGroupIds.map((groupId) => <option key={groupId} value={ groupId }>{ groupNameFromGroup(groupId) }</option>)
```

Comment:
There is a function that looks up the group name from the group (though it defaults to returning "Unknown" if not found instead of just the group id itself)